tests/test_worker.py

```
@@ -164,39 +184,44 @@ def test_handle_error(worker):
-    )
+        samples = list(WORKER_STATE.collect())[0].samples
+        quit_sample = next((s for s in samples if s.labels["state"] == "quit"), None)
+        assert quit_sample is not None
+        assert quit_sample.value == 1.0
+        assert quit_sample.labels["worker_id"] == str(worker.worker_id)
+        assert float(quit_sample.labels["timestamp"]) == pytest.approx(
+            time.time(), rel=1e-3
+        )
 
-    with patch("sys.exit") as _:
+    with patch("sys.exit") as mock_exit:
```

Ruff (0.8.2)

Local variable `mock_exit` is assigned to but never used

Remove assignment to unused variable `mock_exit`

(F841)